x-pack/test/functional/es_archives/endpoint/metadata/api_feature/mappings.json

```diff
@@ -1,33 +1,49 @@
 {
   "type": "index",
   "value": {
-    "aliases": {
-    },
-    "index": "endpoint-agent-1",
+    "aliases": {},
```

Comment:
Using the ecs generated mapping file here instead of the dynamically created one.